src/vulnpy/trigger/cmdi.py

```diff
@@ -8,7 +8,7 @@ def do_os_system(command):
 
 def do_subprocess_popen(command):
     process = subprocess.Popen(
-        command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True
```

Comment:
I'm sure you came across the good arguments for why we should avoid `shell=True` at all costs. However, since we're building a _vulnerable_ application, I don't see an issue with it.

Reply:
hahaha yes of course. `shell` is definitely not a good idea under normal circumstances